src/server/pkg/deploy/assets/assets.go

```diff
@@ -411,7 +427,8 @@ func RethinkRc(backend backend, volume string, hostPath string) *api.Replication
 		spec.Spec.Template.Spec.Volumes[0].PersistentVolumeClaim = &api.PersistentVolumeClaimVolumeSource{
 			ClaimName: rethinkVolumeClaimName,
 		}
-	} else if backend == localBackend {
+	} else if backend == localBackend || backend == microsoftBackend {
+		// ToDo: workaround until https://github.com/pachyderm/pachyderm/issues/960
```

Comment:
Oops. Sorry for the confusion. Yep, workaround until k8s 1.4 + Azure data-disk support (which is the next PR I'll work on)

Actually, not sure if this change is entirely necessary but it's purpose was to use the local host path for the time being.